pkgs/by-name/az/azure-cli/extensions-manual.nix

```diff
@@ -38,6 +38,18 @@
     meta.maintainers = with lib.maintainers; [ obreitwi ];
   };
 
+  application-insights = mkAzExtension rec {
```

Comment:
Please try to keep the extensions in alphabetical order.

Reply:
Done!